flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/enumerator/KafkaEnumeratorTest.java

```diff
@@ -166,7 +167,8 @@ public void testReaderRegistrationTriggersAssignments() throws Throwable {
         }
     }
 
-    @Test(timeout = 30000L)
+    @Test
+    @Timeout(30000L)
```

Comment:
In junit5 default unit is seconds while in junit4 is millis.
Do we really need to set timeout for 30000 secs?